ts/IterableInboxMessageCell.tsx

```diff
@@ -1,47 +1,111 @@
 "use strict"
-import React, { Component } from "react"
-import { View, Text, StyleSheet } from "react-native"
+import React, { Component, useState } from "react"
+import { 
+   View, 
+   Text, 
+   StyleSheet, 
+   TouchableOpacity 
+} from "react-native"
 
 type MessageCellProps = {
+   index: number,
    message: {[key: string]: any},
+   handleMessageSelect: Function,
    last: boolean 
 }
 
-const IterableInboxMessageCell = ({ message, last }: MessageCellProps) => {
+const IterableInboxMessageCell = ({ index, message, handleMessageSelect, last }: MessageCellProps) => {
    const unreadIndicator = "\u2022";
    const messageTitle = message.inboxMetadata.title
    const messageBody = message.inboxMetadata.subtitle
    const messageCreatedAt = message.createdAt
 
+   const [active, setActive] = useState(false);
+
    function displayUnreadMessage() {
```

Comment:
if i can make an architectural suggestion: i would put each of these individual react components into its own file and then import them all.  it's a big re-write, i realize, so i would _also_ suggest not doing that as part of this specific PR, but as a follow up ticket. Potentially even a tech-debt ticket to be picked up on test day. 

The reasons why this separation may be beneficial are:
1. the cognitive load to parse what's going on is much smaller when a file isn't needlessly long. 
2. from an organizational perspective, if in the future, multiple devs need to touch different parts of this file at the same time, there is less chance of merge conflict issues. 
3. (assuming that react-native works like react does) right now, every time that the parent component `IterableInboxMessageCell` is re-rendered because one of it's props changed, all of this code is recompiled again, needlessly.

Reply:
Thanks, Patrick. I will address this on another pull request.